spec/lib/repository_cache_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rspec'
+require_relative '../../lib/repository_cache'
```

Comment:
Why not just `require 'spec_helper'`? Because it takes *forever* to load if you're just BDD-ing a single class as I was here.